packages/support/resources/views/components/link.blade.php

```diff
@@ -67,22 +69,36 @@
         is_string($color) ? "fi-color-{$color}" : null,
     ]);
 
-    $labelClasses = \Illuminate\Support\Arr::toCssClasses([
-        'font-semibold group-hover/link:underline group-focus-visible/link:underline' => ! $labelSrOnly,
-        match ($size) {
-            ActionSize::ExtraSmall => 'text-xs',
-            ActionSize::Small => 'text-sm',
-            ActionSize::Medium => 'text-sm',
-            ActionSize::Large => 'text-sm',
-            ActionSize::ExtraLarge => 'text-sm',
-            default => null,
-        } => ! $labelSrOnly,
-        match ($color) {
-            'gray' => 'text-gray-700 dark:text-gray-200',
-            default => 'text-custom-600 dark:text-custom-400',
-        } => ! $labelSrOnly,
-        'sr-only' => $labelSrOnly,
-    ]);
+    if ($labelSrOnly) {
+        $labelClasses = 'sr-only';
+    } else {
```

Comment:
Can you swap this condition so it's `! $labelSrOnly`, so the important classes are above the `sr-only`

Reply:
Added normal weight only reason for not including it was with it being left out of the `FontWeight` enum. I can't see any harm in adding it though.